code/modules/modular_computers/hardware/nano_printer.dm

```diff
@@ -1,62 +1,71 @@
 /obj/item/weapon/computer_hardware/nano_printer
 	name = "nano printer"
-	desc = "Small integrated printer with scanner and paper recycling module."
+	desc = "Small integrated printer with paper recycling module."
 	power_usage = 50
+	origin_tech = list(TECH_DATA = 2, TECH_ENGINEERING = 2)
 	critical = 0
 	icon_state = "printer"
 	hardware_size = 1
-	var/stored_paper = 5
-	var/max_paper = 10
-	var/obj/item/weapon/paper/P = null	// Currently stored paper for scanning.
+	var/stored_paper = 50
```

Comment:
why are these so high? The bay numbers are fine, printing 50 sheets of paper is ridiculous

Reply:
Thats bays bug, fixed